packages/rspack/src/config/types.ts

```diff
@@ -1073,6 +1073,10 @@ export type JsonParserOptions = {
 	 * The depth of json dependency flagged as `exportInfo`.
 	 */
 	exportsDepth?: number;
+	/**
+	 * If Rule.type is set to 'json' then Rules.parser.parse option may be a function that implements custom logic to parse module's source and convert it to a string.
+	 */
+	parse?: (source: string) => object | string;
```

Comment:
```suggestion
	parse?: (source: string) => any;
```

Reply:
i have a question: why use any instead of keeping it consistent with Webpack?:

https://webpack.docschina.org/configuration/module/#ruleparserparse

Reply:
It seems the doc is outdated https://github.com/webpack/webpack/blob/3919c844eca394d73ca930e4fc5506fb86e2b094/declarations/plugins/JsonModulesPluginParser.d.ts#L15